openfl/component/collaborator/collaborator.py

```diff
@@ -181,7 +194,7 @@ def get_tasks(self):
 
         return tasks, round_number, sleep_time, time_to_quit
 
-    def do_task(self, task, round_number):
+    def do_task(self, task: Union[Task, str], round_number: int) -> None:
```

Comment:
```suggestion
    def do_task(self, task: Task, round_number: int) -> None:
```

`str` is not possible since we call class attributes inside

Reply:
But we have check `isinstance(task, str)` at 210 row